chrome/content/privly-applications

```diff
@@ -1 +1 @@
-Subproject commit 5547688ed5c632bc52755c9b5e5f783875248e91
+Subproject commit 0cd4e24320568c685924418fb27e8e5db7ae6078
```

Comment:
Do you have a summary of changes for the privly-applications? How far from gsoc-develop is it?